libs/ledger/src/tx_status_http_interface.cpp

```diff
@@ -38,7 +38,10 @@ namespace ledger {
 TxStatusHttpInterface::TxStatusHttpInterface(TransactionStatusCache &status_cache)
   : status_cache_{status_cache}
 {
-  Get("/api/status/tx/(digest=[a-fA-F0-9]{64})",
+  Get("/api/status/tx/(digest=[a-fA-F0-9]{64})", "Retrieves a transaction status.",
+      {
+          {"digest", "The transaction hash.", http::validators::StringValue()},
```

Comment:
I agree with you and you will see that they are not actively validating at the moment if you dig deeper into the code. There are two reasons for having them:
1) OpenAPI provides schemas for all parameters 
2) They can provide more stringent validation than regex. For instance, an Address can be validated to check if the address checksum is actually correct.

We could rename them to Schema, rather than validators?